apps/starter/src/ensemble/screens/home.yaml

```diff
@@ -38,3 +43,7 @@ ViewGroup:
     footer:
       Text:
         text: Give us feedback
+    onCollapse:
```

Comment:
Sorry, this should have been clarified in the ticket more. Instead of `onCollapse`, we should support something like
```
Button:
  onTap:
    mySidebarMenu.collapse()
```

Basically  expose a way to collapse the menu programmatically so users can put their own UI over it also